hypershift-operator/controllers/nodepool/nodepool_controller_test.go

```diff
@@ -3097,6 +3097,18 @@ func TestIsArchAndPlatformSupported(t *testing.T) {
 			},
 			expect: true,
 		},
+		{
```

Comment:
Can we add another case for arm for none platform and maybe an invalid case for the none platform please?

Reply:
Hi @bryan-cox, added the suggested tests. Please review. Thank you.